Modelica/UsersGuide/ReleaseNotes.mo

```diff
@@ -130,89 +130,363 @@ class Version_4_1_0 "Version 4.1.0 (Month D, 20YY)"
+</tr>
+<tr>
+<td colspan=\"2\"><h4>Modelica.Mechanics.Rotational.Sources</h4></td>
+</tr>
+<tr>
+<td><p>InverseSpeedDependentTorque</p></td>
+<td><p>Model of inverse speed-dependent torque.</p></td>
+</tr>
+<tr>
+<td colspan=\"2\"><h4>Modelica.Mechanics.Translational.Sources</h4></td>
+</tr>
+<tr>
+<td><p>InverseSpeedDependentForce</p></td>
+<td><p>Model of inverse speed-dependent force.</p></td>
+</tr>
+<tr>
+<td colspan=\"2\"><h4>Modelica.Units.SI</h4></td>
+</tr>
+<tr>
+<td><p>StoichiometricCoefficient</p></td>
+<td><p>&nbsp;New SI unit type </p></td>
+</tr>
 </table>
-
-<p><br>
-The following <font color=\"blue\"><strong>existing components</strong></font> have been <font color=\"blue\"><strong>improved</strong></font> in a <font color=\"blue\"><strong>backward compatible</strong></font> way:
-</p>
-
-<table border=\"1\" cellspacing=\"0\" cellpadding=\"2\" style=\"border-collapse:collapse;\">
-<tr><td colspan=\"2\"><strong>Modelica.Blocks.Sources</strong></td></tr>
-<tr><td>CombiTimeTable</td>
-    <td>Added support of reading CSV files.</td></tr>
-<tr><td colspan=\"2\"><strong>Modelica.Blocks.Tables</strong></td></tr>
-<tr><td>CombiTable1Ds<br>CombiTable1Dv<br>CombiTable2Ds<br>CombiTable2Dv</td>
-    <td>Added support of reading CSV files.</td></tr>
-<tr><td colspan=\"2\"><strong>Electrical.PowerConverters.DCDC</strong></td></tr>
-<tr><td>HBridge</td>
-    <td>An enhanced distribution of the fire signals avoids a short circuit on the source, and enables an enhanced pwm algorithm.</td></tr>
-<tr><td>Control.SignalPWM</td>
-    <td>The reference signal can be chosen between sawtooth and triangle, and 
-        the comparison between dutyCycle and reference signal is either applied common or separated for both fire ports.</td></tr>
-<tr><td colspan=\"2\"><strong>Mechanics.Rotational.Components</strong></td></tr>
-<tr><td>BearingFriction</td>
-    <td>The table interpolation in <code>tau_pos</code> utilizes the interpolation based on <a href=\"modelica://Modelica.Blocks.Types.ExternalCombiTable1D\">ExternalCombiTable1D</a>.</td></tr>
-<tr><td>LossyGear</td>
-    <td>The table interpolation in <code>lossTable</code> utilizes the interpolation based on <a href=\"modelica://Modelica.Blocks.Types.ExternalCombiTable1D\">ExternalCombiTable1D</a>.</td></tr>
-<tr><td colspan=\"2\"><strong>Mechanics.Translational.Components</strong></td></tr>
-<tr><td>SupportFriction</td>
-    <td>The table interpolation in <code>f_pos</code> utilizes the interpolation based on <a href=\"modelica://Modelica.Blocks.Types.ExternalCombiTable1D\">ExternalCombiTable1D</a>.</td></tr>
+<p><br><br>The following <strong><font color=\"blue\">existing components</font></strong> have been <strong><<font color=\"blue\">improved</font></strong> in a <strong><font color=\"blue\">backward compatible</font></strong> way: </p>
+<table cellspacing=\"0\" cellpadding=\"2\" border=\"1\"><tr>
+<td colspan=\"2\"><h4>Modelica.Blocks.Sources</h4></td>
+</tr>
+<tr>
+<td><p>CombiTimeTable</p></td>
```

Comment:
Please do not use <p> environment within <td>. Even if it is maybe possible in html, I don't know any such usage in MSL documentation.

Reply:
We haven't use such formating so far. See https://github.com/modelica/ModelicaStandardLibrary/blob/0d88f6b534760f4e25e3072db1be4973b4003c91/Modelica/UsersGuide/Conventions.mo#L418

Reply:
To me, the fact that we are not explicitly forbidding the use of <p> tags inside tables doesn't meant we can't use that. Anyway, I can remove them, no problem.

Reply:
Done in 4a14187b